app/assets/javascripts/short-form/ShortFormNavigationService.js.coffee

```diff
@@ -111,6 +111,11 @@ ShortFormNavigationService = ($state) ->
           'household-members'
         else
           'household-intro'
+      when 'general-lottery-notice'
+        'status-programs'
+      when 'status-vouchers'
```

Comment:
@chrisdolendo I went with your idea and refactored `preferencesApplyForHousehold` to be in `ShortFormApplicationService`... however then I realized this function is in `ShortFormNavigationService`. But, it's totally fine for services to include each other, and in fact it probably makes sense for `ShortFormNavigationService` to have access to the AppService as well (like for example it's sort of unnecessary to pass `$scope.application` as a param to `Service.backPageState` when it could just get that directly from the AppService, we could refactor that). Likewise I realize I've put in some "navigation-y" features in AppService because they also needed access to AppService functions, like `authorizedToProceed` uses `userCanAccessSection` -- however if NavService can tap into these functions then problem solved!
Long story short -- this works now!